webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js

```diff
@@ -224,6 +224,8 @@ export class SakaiRubric extends RubricsElement {
 
   toggleRubric(e) {
 
+    console.log("HERERE");
```

Comment:
Please remove this and the others 2 console.log(). Thanks!

Reply:
Lol. Good spot :)